README.md

```diff
@@ -1 +1,185 @@
-# Portfolio_setup_and_mobile_first
\ No newline at end of file
+<div align="center">
+  
+  <br/>
+
+  <h3><b>Microverse README Template</b></h3>
+
```

Comment:
- [ ] Great work customizing your README file with information related to this project!💯 However, please kindly make sure to remove the Microverse README Template because this is your personal portfolio project. 🤓